src/components/Pages/Unit/widgets/Downloads/MediaDownloads.js

```diff
@@ -48,7 +48,8 @@ class MediaDownloads extends Component {
     language: PropTypes.string.isRequired,
     contentLanguage: PropTypes.string.isRequired,
     t: PropTypes.func.isRequired,
-    displayDivider: PropTypes.bool
+    displayDivider: PropTypes.bool,
+    chronicles: PropTypes.shape(),
```

Comment:
I'd expect to see shape in `PropTypes.shape()`. Sorry that I missed it in prev instances of chronicles...

Reply:
Fixed for callback.